### PR TITLE
analyze view fixed

### PR DESCRIPTION
**Pull request by bergano65**

### CODE REVIEW

File: CogsMinimizer/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
         if (connTimeHours <= 24)
         {
-            alert("This subscription has not been analysed yet. Please check back tomorrow. If it’s already tomorrow, then contact support.");
+            alert("This subscription has not been analyzed yet. Please check back tomorrow. If it’s already tomorrow, then contact support.");
```
Comment (shklar) [inline CogsMinimizer/Views/Home/Index.cshtml]: Replace:
"If it's already..."
with
"If this problem persists, please contact support."

File: CogsMinimizer/Views/Subscription/Analyze.cshtml [hunk 2]
```diff
@@ -72,6 +77,34 @@
...
+         $(statusBtn).addClass("btn");
+         $(statusBtn).addClass("tbl-btn");
+         $(statusBtn).addClass("btn-info");
+         $(statusBtn).addClass("btn-xs");
+
+         if (status == "Valid")
+         {
+             $(statusBtn).addClass("glyphicon-ok");
+         }
+         else if (status == "Expired") {
```
Comment (shklar) [inline CogsMinimizer/Views/Subscription/Analyze.cshtml]: These lines are repeated

**Comment by bergano65** [comment]
What about:

    function CheckSubscriptionConnectDate(btn)
    {
        var href = $(btn).prop("href");
        var connTimeHoursMatches = href.match(/ConnectedTime=([0-9]*)/);
        var connTimeHours = connTimeHoursMatches[1]

        if (connTimeHours <= 24)
        {
            alert("After starting monitoring subscription is analyzed daily by external job. This subscription probably has not been analysed yet since started monitoring " + connTimeHours + "  hours ago. Please check back tomorrow. If tomorrow no resources will be discovered, then contact support.");
        }
    }

**Comment by shklar** [comment]
No need for something that elaborate.
Just say as noted below.

This subscription has not been analyzed yet. Please check back tomorrow. If this problem persists, please contact support.

**Comment by bergano65** [comment]
I would say from not extended comment nobody understand

-          That error is probable, probably subscription already analyzed

-           How to determine problem still exist.



Let’s say user opens fresh monitored subscription for analyzing, see resources (happily job already run), but was warned before getting view shown.

We write “ subscription isn’t analyzed yet.”. but tomorrow how he will determine there wasn’t problem at all. We write “if problem persist” ). Which problem? How user will understand persist problem or not? He saw resources already yesterday he see them now.

My variant explains everything )

**Comment by bergano65** [comment]
What do you think?

From: Evgeny Vitenberg
Sent: Tuesday, December 27, 2016 4:40 AM
To: 'shklar/SubMinimizer' <reply@reply.github.com>; shklar/SubMinimizer <SubMinimizer@noreply.github.com>
Cc: Author <author@noreply.github.com>
Subject: RE: [shklar/SubMinimizer] analyze view fixed (#54)

I would say from not extended comment nobody understand

-          That error is probable, probably subscription already analyzed

-           How to determine problem still exist.



Let’s say user opens fresh monitored subscription for analyzing, see resources (happily job already run), but was warned before getting view shown.

We write “ subscription isn’t analyzed yet.”. but tomorrow how he will determine there wasn’t problem at all. We write “if problem persist” ). Which problem? How user will understand persist problem or not? He saw resources already yesterday he see them now.

My variant explains everything )

**Comment by shklar** [comment]
I think the comment is clear enough. We have Harel's feedback as well.
If we encounter more questions we can always change it.

Sent from my Windows Phone
________________________________

**Comment by bergano65** [comment]
Pls check corrections and statistics added to  main view also. As it’s few lines decided join change with analyze view changes

**Pull request merged**